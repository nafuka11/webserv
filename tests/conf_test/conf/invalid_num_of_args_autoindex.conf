server {
    location / {
        autoindex on off;
    }
}