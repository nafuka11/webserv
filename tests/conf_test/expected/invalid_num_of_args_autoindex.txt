webserv: invalid number of arguments in "autoindex" directive in ./conf/invalid_num_of_args_autoindex.conf:3
